automated-tests/src/test/resources/cucumber/settings/scope.feature

```diff
@@ -0,0 +1,123 @@
+################################################################################
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+Feature: In order to manage Scopes
+
+  Background: 
+    Given an instance of OneView
+      And OneView credentials located in "src/test/resources/oneView.properties"
+      And an instance of Scope
+
+  @create
+  Scenario: Creation of a new Scope
+    Given Resource values as follows:
+      | name        | BDD Scope             |
+      | description | BDD Scope description |
+    When OneView runs Resource creation
+      And OneView gets Resource by Name
+    Then I get an ID
+
+  @getAll
+  Scenario: Get all Scopes
+    When OneView lists all
+    Then I get a count
+
+  @get
+  Scenario: Get a Scope by Name
+    Given name "BDD Scope" for Resource
+    When OneView gets Resource by Name
+    Then I get an ID
+
+  @get
+  Scenario: Get a Scope by Id
+    Given name "BDD Scope" for Resource
+    When OneView gets Resource by Name
+      And OneView gets Resource by ID
+    Then I get a Resource Name
+
+  @create
+  Scenario: Creation of a new FC Network
+    Given an instance of FC Network
+      And Resource values as follows:
+      | name                    | fc-network-bdd |
+      | fabricType              | FabricAttach   |
+      | linkStabilityTime       |             30 |
+      | autoLoginRedistribution | true           |
+    When OneView runs Resource creation
+      And OneView gets Resource by Name
+    Then I get an ID
+
+  @create
+  Scenario: Creation of a new Ethernet Network
+    Given an instance of Ethernet Network
+      And Resource values as follows:
+      | name         | net-network-bdd |
+      | ethernetType | Tagged          |
+      | vlanId       |        301      |
+      | purpose      | General         |
+      | private      | false           |
+      | smartLink    | true            |
+      And bandwidth values as follows:
+      | maxBandwidth     | 1000 |
+      | typicalBandwidth | 1000 |
+    When OneView runs Resource creation
+      And OneView gets Resource by Name
+    Then I get an ID
+
+  @patch
+  Scenario: Update a Scope by Patch
+    Given name "BDD Scope" for Resource
+      And Resource values will be updated as follows:
+      | fcNetwork       | fc-network-bdd  |
```

Comment:
Keep a more general name like resourceName, instead of hardcoding fcNetwork, because the user might want to update scope for a different resource type in the future.